orcid-model/src/main/java/org/orcid/jaxb/model/message/Locale.java

```diff
@@ -81,7 +81,10 @@
     IT("it"),
     
     @XmlEnumValue("ja")
-    JA("ja");
+    JA("ja"), 
+    
+    @XmlEnumValue("xx")
```

Comment:
Hey @wjrsimpson @rcpeters please look at this, this is the fastest way to allow a user to have the X language in his locale.

Is this ok?

Thanks